### PR TITLE
Changelog 20.1

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Non-significant (changelog entry is not required)

### CODE REVIEW

**Comment by alesapin** [comment]
Just script output at this moment.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,338 @@
...
+* Fixed potential performance issue in "Context::getTable" method. [#8538](https://github.com/ClickHouse/ClickHouse/pull/8538) ([alexey-milovidov](https://github.com/alexey-milovidov))
+* Improved decoding performance of DoubleDelta and Gorilla codecs by (TODO: edit after performance tests are done) roughly 30-40%. ... [#8019](https://github.com/ClickHouse/ClickHouse/pull/8019) ([Vasily Nemkov](https://github.com/Enmk))
+* Improved performance of base64 related functions. [#8444](https://github.com/ClickHouse/ClickHouse/pull/8444) ([alexey-milovidov](https://github.com/alexey-milovidov))
+* Added a function `geoDistance`. It is similar to `greatCircleDistance` but uses approximation to WGS-84 ellipsoid model. The performance of both functions are near the same. [#8086](https://github.com/ClickHouse/ClickHouse/pull/8086) ([alexey-milovidov](https://github.com/alexey-milovidov))
+* Faster mix/max aggregtion functions for Decimals [#8144](https://github.com/ClickHouse/ClickHouse/pull/8144) ([Artem Zuikov](https://github.com/4ertus2))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Typo.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,300 @@
...
+* Limit maximum number of streams for read from `StorageFile` and `StorageHDFS`. Fixes https://github.com/ClickHouse/ClickHouse/issues/7650. [#7981](https://github.com/ClickHouse/ClickHouse/pull/7981) ([alesapin](https://github.com/alesapin))
+* Fix bug in `ALTER ... MODIFY ... CODEC` query, when user specify both default expression and codec. Fixes [8593](https://github.com/ClickHouse/ClickHouse/issues/8593). [#8614](https://github.com/ClickHouse/ClickHouse/pull/8614) ([alesapin](https://github.com/alesapin))
+* Fix error in background merge of columns with `SimpleAggregateFunction(LowCardinality)` type. [#8613](https://github.com/ClickHouse/ClickHouse/pull/8613) ([Nikolai Kochetov](https://github.com/KochetovNicolai))
+* Fixed type check in function `toDateTime64`. [#8375](https://github.com/ClickHouse/ClickHouse/pull/8375) ([Vasily Nemkov](https://github.com/Enmk))
+* Now server do not crash on `LEFT` or `FULL JOIN` wittable and Join engine and unsupported `join_use_nulls` settings. [#8479](https://github.com/ClickHouse/ClickHouse/pull/8479) ([Artem Zuikov](https://github.com/4ertus2))
```
Comment (4ertus2) [inline CHANGELOG.md]: with Join engine

**Review by alexey-milovidov — APPROVED** [review]
Ok but will edit.

**Comment by den-crane** [comment]
Connected to ClickHouse server version 20.1.2 revision 54431.

There is no max_threads_for_parallel_parsing 

SET input_format_parallel_parsing = 1
Ok.

SET max_threads_for_parallel_parsing = 1

Received exception from server (version 20.1.2):
Code: 115. DB::Exception: Received from localhost:9000. DB::Exception: Unknown setting max_threads_for_parallel_parsing.

**Pull request merged**